### DISCUSSION TITLE
private and public list's

### DISCUSSION THREAD

**Discussion by S7TN** [community/community#37191] [Other Feature Feedback, Questions, & Ideas]
Hello, we want to put private and public menus like repositories

If you like this give a like & Follow (: 

[My iG](https://instagram.com/hxn.ops))

[My TG](https://t.me/HEXiN1K)

@github-staff 
@HXN2
@github
@Community

**Comment by S7TN** [comment]
Like

**Comment by ossanna16** [comment]
Hi there @hwansday and welcome to our community! Thank you for your post. Can you please elaborate on your suggestion a bit further?

**Reply by S7TN** [reply]
Like the repo, we can make it private and public, and we also want the list to be private and public

**Comment by serioder5** [comment]
Я новенькая

**Reply by S7TN** [reply]
Like the repo, we can make it private and public, and we also want the list to be private and public

**Comment by jfullstackdev** [comment]
you can have the list as public but if you add private repo, then boomm ... no one can see it except you and those added as contributors, 

would that not be enough ?

**Reply by S7TN** [reply]
I mean, I want it especially for me only, for example, warehouses

**Reply by jfullstackdev** [reply]
yup, it's all yours, the only thing that can be seen is the List itself but there is nothing inside there

actually it's good too, for example in a list, you want some public and private repos, then others will only see those public repos not the private ones

**Reply by Hillside3000** [reply]
https://media.gcflearnfree.org/content/57c0530bf244e80ed8f383f6_08_26_2016/recover_password_address.png

**Comment by dithmer** [comment]
I really like lists as an in-built collection for projects and tech stacks or similar

private lists would be amazing to collect repositories for e.g. work projects, work techstack etc., when your employer don't want those things to be public.

Definetly seeing forward to it!

**Reply by dithmer** [reply]
For sure it is possible to organize those things elsewhere in a document or something, but why not using features of github, which already exist?

**Reply by S7TN** [reply]
We just want it to be private

**Comment by DanielNewsGlobal** [comment]
Ciao, sò purtatu à ricunsiderà a riverberazione di brooding ripetendu ciò chì Diu hà fattu.
